File: proj_2/app/src/main/java/com/example/proj_2/tab2/MediaFileData.java
package com.example.proj_2.tab2;

import android.net.Uri;

import java.util.Date;

public class MediaFileData {
    public Long bucketId;
    public Long id;
    public Date dateTaken;
    public String displayName;
    public Uri uri;
    public String bucketName;

    public MediaFileData(Long bucketId, Long id, Date dateTaken, String displayName, Uri uri, String bucketName){
        this.bucketId = bucketId;
        this.id = id;
        this.dateTaken = dateTaken;
        this.displayName = displayName;
        this.uri = uri;
        this.bucketName = bucketName;
    }


}

File: ji/project1_final/app/src/main/java/com/example/project1_final/model/MediaFileData.kt
package com.example.project1_final.model

import android.net.Uri
import java.util.*

data class MediaFileData(
    val id:Long,
    val dateTaken: Date,
    val displayName: String,
    val uri: Uri,
    val bucketId: Long,
    val bucketName : String
)